### ISSUE TITLE
Open specific folders after SSH connection

### ISSUE THREAD

**Issue by scaliaven** [thebaselab/codeapp#1146]
Hi, great work here! Currently, I could only open the home directory after ssh connection. Is it possible to advance ssh features so that one can open up specific folders after connection? (easier to jump between folders in different paths)

**Comment by bummoblizard** [comment]
Yes. You can tap and hold any folder and select `Assign as workspace folder`.

**Comment by scaliaven** [comment]
Hi, thanks for your reply. I think I need to clarify my problem a bit. I could use the resolving home path function, but the home directory is not where I put most of my stuff. The option left for me is to open folders starting from the root directory. However, I am using my university's HPC and there are many accounts. To tap and hold my folders, I need to first open up a directory that contains all the accounts and find mine. It is very difficult to open up such a large directory.

**Comment by scaliaven** [comment]
Currently, the issue is resolved by creating symbolic links that link to my other directories from home. But, I still think that open up folders directly by searching their path is a feature that you guys could try to enhance.
